.circleci/config.yml

```diff
@@ -18,25 +18,73 @@ aliases:
         command: ./scripts/cibuild
     - save_cache: *save_sbt_cache
 
+  - &run_cipublish
+    - checkout
+    - run: echo "${SCALA_VERSION}" > /tmp/scala_version
+    - restore_cache: *restore_sbt_cache
+    - run:
+        name: "Import signing key"
```

Comment:
Why not handle this in `cipublish`?

Reply:
My thinking was that this is a CircleCI-specific step that we wouldn't use in Travis or Jenkins, etc.

Reply:
Fair.